marketplace-and-integrations/integrations/commercetools.md

```diff
@@ -0,0 +1,42 @@
+---
+description: >-
+  Details an integration available for CommerceTools, built and maintained by Umbraco HQ.
+---
+
+# CommerceTools
+
+This integration provides a product and category picker, with data sourced from a [CommerceTools](https://commercetools.com/) installation.
+
+## Package Links
+
+- [NuGet install](https://www.nuget.org/packages/Umbraco.Cms.Integrations.Commerce.CommerceTools)
+- [Source code](https://github.com/umbraco/Umbraco.Cms.Integrations/tree/main/src/Umbraco.Cms.Integrations.Commerce.CommerceTools)
+- [Umbraco marketplace listing](https://marketplace.umbraco.com/package/umbraco.cms.integrations.commerce.commercetools)
+
+## Prerequisites
+
+Requires minimum version of Umbraco CMS: 8.5.4.
```

Comment:
Is this package only available for Umbraco 8?

Reply:
It is yes.  I was one of the first ones done and then priorities changed which meant it didn't get taken forward.